src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -4233,15 +4282,365 @@ module Mutations = struct
+            [ ( "ledger"
+              , `List (List.map (Ledger.to_list ledger) ~f:Account.to_yojson) )
+            ] ;
+        wait_until_zkapps_deployed ~deployed:true ~mina ~ledger
+          ~fee_payer_keypair ~account_creator_keypair ~constraint_constants
+          ~logger ~uuid ~stop_signal ~stop_time keypairs
+
+    let schedule_zkapp_commands =
+      io_field "scheduleZkappCommands"
         ~args:
           Arg.
-            [ arg "handle" ~doc:"Payment scheduler handle"
+            [ arg "input" ~doc:"Zkapp commands details"
+                ~typ:(non_null Types.Input.Itn.ZkappCommandsDetails.arg_typ)
+            ]
+        ~typ:(non_null string)
+        ~resolve:(fun { ctx = mina; _ } () input ->
+          return
+          @@
+          match input with
+          | Error err ->
+              Error (sprintf "Invalid input to zkapp command scheduler: %s" err)
+          | Ok zkapp_command_details -> (
+              if List.is_empty zkapp_command_details.fee_payers then
+                Error "Empty list of fee payers"
+              else
+                let logger = Mina_lib.top_level_logger mina in
+                let uuid = Uuid.create_random Random.State.default in
+                let ivar = Ivar.create () in
+                ( match Uuid.Table.add scheduler_tbl ~key:uuid ~data:ivar with
+                | `Ok ->
+                    ()
+                | `Duplicate ->
+                    failwith
+                      "Unexpected duplicate scheduled zkApp commands handle" ) ;
+                let wait_span =
+                  1. /. zkapp_command_details.transactions_per_second
+                  |> Time.Span.of_sec
+                in
+                let duration_span =
+                  Time.Span.of_min
+                    (Float.of_int zkapp_command_details.duration_in_minutes)
+                in
+                let tm_start = Time.now () in
+                let tm_end = Time.add tm_start duration_span in
+                match get_ledger_and_breadcrumb mina with
+                | None ->
+                    Error "Could not get best tip ledger"
+                | Some (ledger, _best_tip) -> (
+                    [%log info] "Starting zkApp scheduler with handle %s"
+                      (Uuid.to_string uuid) ;
+                    let { Precomputed_values.constraint_constants; _ } =
+                      (Mina_lib.config mina).precomputed_values
+                    in
+                    let zkapp_account_keypairs =
+                      List.init zkapp_command_details.num_of_zkapp_accounts
+                        ~f:(fun _ -> Signature_lib.Keypair.create ())
+                    in
+                    let unused_keypairs =
+                      List.init
+                        (20 + zkapp_command_details.generating_new_accounts)
+                        ~f:(fun _ -> Signature_lib.Keypair.create ())
+                    in
+                    let account_creator_keypair =
```

Comment:
Let's have multiple account creator keypairs used in round-table fashion with interval specified as a parameter